input/core/src/commonMain/kotlin/majestic/Tag.kt

```diff
@@ -15,9 +15,9 @@ import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 
-fun Modifier.tag() = this
+fun Modifier.tag(color: Color = Color(0xFF5C6BC0).copy(alpha = 0.2f)) = this
```

Comment:
Unnecessary copy call can be embedded directly into the hexadecimal

```suggestion
fun Modifier.tag(color: Color = Color(0x335C6BC0)) = this
```